proof/refine/AARCH64/Refine.thy

```diff
@@ -669,10 +669,13 @@ lemma entry_corres:
       apply (rule corres_split)
          apply simp
          apply (rule threadset_corresT)
-            apply (simp add: tcb_relation_def arch_tcb_relation_def
-                             arch_tcb_context_set_def atcbContextSet_def)
-           apply (clarsimp simp: tcb_cap_cases_def cteSizeBits_def)
-          apply (clarsimp simp: tcb_cte_cases_def cteSizeBits_def)
+               apply (simp add: tcb_relation_def arch_tcb_relation_def
+                                arch_tcb_context_set_def atcbContextSet_def)
+              apply (clarsimp simp: tcb_cap_cases_def cteSizeBits_def)
+             apply (clarsimp simp: tcb_cte_cases_def cteSizeBits_def)
+            apply simp
+           apply simp
+          apply simp
```

Comment:
this is similar to a `; simp` pattern I mentioned before (same rule, `threadset_corresT` I think)